drivers/adc/ad719x/ad719x.c

```diff
@@ -413,9 +413,14 @@ int ad719x_channels_select(struct ad719x_dev *dev,
 		}
 	}
 
-	return ad719x_set_masked_register_value(dev, AD719X_REG_CONF,
-						AD719X_CONF_CHAN(0x3FF), AD719X_CONF_CHAN(chn_mask),
-						3);
+	if (dev->chip_id == AD7194)
+		return ad719x_set_masked_register_value(dev, AD719X_REG_CONF,
+							NO_OS_GENMASK(15,8), chn_mask,
```

Comment:
something looks wrong here, chn_mask is 0x1FF... which does not look correct seeing how you want to mask starting from bit 8. you should use AD719X_CONF_CHAN(chn_mask) and instead of NO_OS_GEN_MASK you can similarly use AD719X_CONF_CHAN(0x1FF)